plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -228,13 +228,20 @@ void chain_plugin::set_program_options(options_description& cli, options_descrip
          ("blocks-dir", bpo::value<bfs::path>()->default_value("blocks"),
           "the location of the blocks directory (absolute path or relative to application data dir)")
          ("blocks-log-stride", bpo::value<uint32_t>()->default_value(config::default_blocks_log_stride),
-         "split the block log file when the head block number is the multiple of the split factor")
+         "split the block log file when the head block number is the multiple of the split factor\n"
+         "When the stride is reached, the current block log and index will be renamed 'blocks-num_begin-num_end.log/index'\n"
```

Comment:
Can we use 'block-<num begin>-<num end>.log/index' to make it clearer? (here and in the PR documentation)

Reply:
sorry, markup removed my text
`block-<start num>-<end num>.log/index`